data/presets/leisure/bathing_place.json

```diff
@@ -0,0 +1,45 @@
+{
+    "aliases": [
+        "Bathing spot",
+        "Dip spot",
+        "Dipping spot",
+        "Swim spot",
+        "Swimming spot",
+        "Wild swimming spot"
+    ],
+    "icon": "fas-water-ladder",
```

Comment:
"ladder" does not sound ideal for a place that is often unofficial and will likely not have a ladder. What would be other options? 

Do we use ladder for other presets as well? (Might this cause confusion?)